doc/UsersGuide/source/Configurations.rst

```diff
@@ -283,30 +399,296 @@ Rapid Refresh Forecast System (RRFS)
 
 The RRFS configurations use an :term:`ATM`-only configuration on a high-resolution 
 regional grid with data assimilation capabilities. 
-These tests use the default values set in the ``export_fv3``, ``export_rap_common``, ``export_rrfs_v1``, and/or ``export_hrrr_conus13km`` functions of ``default_vars.sh`` unless other values are explicitly set in a given test file. In all tests, the values in ``export_fv3`` are set first. Depending on the test, some of these values may be overriden by ``export_rrfs_v1`` (which includes values from ``export_rap_common``) or ``export_hrrr_conus13km``. :numref:`Table %s <rrfs-default-vars-comparison>` compares the values set in ``export_fv3`` to the values set in the other functions. 
-
-.. note:: 
-
-   ``export_rrfs_v1`` calls ``export_rap_common``, which calls ``export_fv3``. Values from ``export_fv3`` are set first, followed by values in ``export_rap_common`` and then values in ``export_rrfs_v1``. Values in italics indicate that the value is inherited from a previously-called function. 
+These tests use the default values set in the ``export_fv3`` function of ``default_vars.sh`` unless other values are explicitly set. 
 
-.. _rrfs-default-vars-comparison:
+Current RRFS regression tests cover a wide variety of functionality and involve several 
+physics tests. :numref:`Table %s <rrfs-rts>` contains RTs for RRFS functionality. 
 
-.. csv-table:: *RRFS Default Variables*
-   :file: tables/RRFSDefaultVariables.csv
-   :widths: 50 10 10 10 10
-   :header-rows: 1
-   :stub-columns: 1
+.. attention:: 
+   
+   Certain physics-related settings are common to all of the supported RRFS configurations. These values are set in each test's configuration file because they differ from the ``default_vars.sh`` values:
 
-Current RRFS regression tests cover a wide variety of functionality and involve several 
-physics tests. :numref:`Table %s <rrfs-rts>` (below) contains a selection of RTs for RRFS functionality. Blanks indicate that the value comes from the default setting file. These default values are listed in :numref:`Table %s <rrfs-default-vars-comparison>` above. 
+      * **Set to FALSE:** DO_SAT_ADJ, HYBEDMF, DO_DEEP, SHAL_CNV, LHEATSTRG
+      * **Set to TRUE:** DO_MYNNEDMF, DO_MYNNSFCLAY
+      * **Set to VALUE:** DNATS=0, IALB=2, IEMS=2, IMFSHALCNV=-1, IMFDEEPCNV=-1
+   
+   The "Detailed Physics Parameters" column in :numref:`Table %s <rrfs-rts>` details physics settings that differ from both the ``default_vars.sh`` values and these RRFS-specific defaults. 
+   
 
 .. _rrfs-rts:
 
-.. csv-table:: *RRFS regression test descriptions*
-   :file: tables/rrfs-rts.csv
-   :widths: 20 20 30 50 10 10 10
+.. list-table:: *RRFS regression test descriptions*
+   :widths: 50 10 30 50 10 10 10 10 10
```

Comment:
This list-table is an old version. The CSV table is the right one. Sorry I didn't notice the first time. 
```suggestion
.. csv-table:: *RRFS regression test descriptions*
   :file: tables/rrfs-rts.csv
   :widths: 20 20 30 50 10 10 10
```